modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc

```diff
@@ -11,9 +11,17 @@ Here are some examples:
 [cols=">10%,^25%,65%",options="header",stripes=even]
 |===
 |Version
-|Can Upgrade to 10.3.2?
+|Can Upgrade to 10.5.0?
 |Requirements
 
+|10.4.1
+|Yes
+|
+
+|10.3.1
```

Comment:
```suggestion
|10.3.2
```

Reply:
Can you explain why you want that change? :D

simply suggesting a change without explanation is weird.

Reply:
The other examples in the list have the latest patch release as the example.